python/proton/_data.py

```diff
@@ -705,6 +705,7 @@ def rewind(self):
         the first node.
         """
         assert self._data is not None
+        # print(f"log _data 708: data {self._data}")
```

Comment:
If you have recent enough Python version, you can use `print(f"708: {self._data=}")`. If you add the equal sign, Python will format the print as `708: self._data=the-value`. Cool trick, shows you keep up with Python features ;)

Reply:
That's very useful trick